review/fileToReview.js

```diff
@@ -1,73 +1,119 @@
 
-var domain = "bank.local.fr"
+const winston = require('winston')
+
+const logger = winston.createLogger({
+  defaultMeta: { service: 'bankin-test' },
+  transports: [
+    new winston.transports.Console({
+      format: winston.format.json(),
+    })
+  ],
+})
+
+const domain = 'bank.local.fr'
```

Comment:
Prefer to use const than var. Also i would prefer to store domain as environment variable.